android/CouchbaseLite/src/main/java/com/couchbase/lite/DatabaseConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package com.couchbase.lite;
+
+import android.content.Context;
+
+public class DatabaseConfiguration extends BaseDatabaseConfiguration{
+    private Context context;
+
+    public DatabaseConfiguration() {
```

Comment:
`Database(String name)` is not allowed for Android. But `DatabaseConfiguration()` should be allowed because the user can set directory.